tests/api2/test_005_interface.py

```diff
@@ -99,6 +100,6 @@ def test_001_configure_interface(request, ws_client, get_payload):
             assert c.call('failover.call_remote', 'core.ping') == 'pong'
 
         # it's very important to set this because the `tests/conftest.py` config
-        # (that pytest uses globally for the entirety of CI runs) will check this
-        # value and use the proper IP address (the VIP) on HA systems
-        os.environ['USE_VIP'] = 'YES'
+        # (that pytest uses globally for the entirety of CI runs) uses this IP
+        # address and so we need to make sure it uses the VIP on HA systems
+        host().ip = os.environ['virtual_ip']
```

Comment:
Because the IP is dynamic and has to change based on where we're at in the CI run. `os.environ` is technically a function that returns mutable global variables and we've been doing that since forever. How would you recommend using the global IP without having to touch every function in our test suite?

Reply:
I can use the `vip` variable from `auto_config` if that's what you mean?